src/state/pipeline-run/index.js

```diff
@@ -0,0 +1,6 @@
+import { makeLocalGetter } from '../../utils/object';
+import get from 'lodash/get';
+
+const localGetter = makeLocalGetter('pipelineRun');
+
+export const getPipelineRun = (state) => get(state, 'pipelineRun', null);
```

Comment:
remove unused getter
```suggestion
import get from 'lodash/get';

export const getPipelineRun = (state) => get(state, 'pipelineRun', null);
```